contrib/for-tests/hostdir-mount-tester/mt-hostdir.go

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright 2015 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+)
+
+var (
+	fileModePath        = ""
+	readFileContentPath = ""
+	writeNewFilePath    = ""
+)
+
+func init() {
+	flag.StringVar(&fileModePath, "file_mode", "", "Path to print the filemode of")
```

Comment:
Just a habit.
See also
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cmd/e2e/e2e.go#L46
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/contrib/for-tests/mount-tester/mt.go#L35